assets/src/apps/delivery/layouts/deck/DeckLayoutFooter.tsx

```diff
@@ -249,7 +249,7 @@ const DeckLayoutFooter: React.FC = () => {
       currentFeedbacks?.length > 0 &&
       displayFeedbackIcon
     ) {
-      if (currentPage.custom?.advancedAuthoring && !currentPage.custom?.allownavigation) {
+      if (currentPage.custom?.advancedAuthoring && !currentPage.custom?.enableHistory) {
```

Comment:
for this one we might need to selectEnableHistory instead of testing the page property